nixos/modules/services/backup/bupstash.nix

```
@@ -0,0 +1,296 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (lib)
+    literalExpression
+    mapAttrs'
+    nameValuePair
+    mkEnableOption
+    mkIf
+    mkMerge
+    mkOption
+    optional
+    optionalString
+    ;
+
+  inherit (lib.types)
+    attrsOf
+    either
+    lines
+    listOf
+    nullOr
+    package
+    passwdEntry
+    path
+    str
+    submodule
+    ;
+
+  cfg = config.services.bupstash;
+
+  bupstash = "${cfg.package}/bin/bupstash";
+  defaultUser = "bupstash-server";
+in
+{
+  meta.maintainers = [ lib.maintainers.thubrecht ];
+
+  options.services.bupstash = {
+    enable = mkEnableOption "bupstash backup service.";
```

```suggestion
    enable = mkEnableOption "bupstash backup service";
```

The `.` will be added automatically.